simple-wp-helpdesk/assets/src/theme-violet/tokens.scss

```
@@ -0,0 +1,129 @@
+    --swh-radius-sm: 4px;
+    --swh-radius-md: 8px;
+    --swh-radius-lg: 12px;
+    --swh-radius-full: 999px;
+
+    /* Shadows with primary tint */
+    --swh-shadow-sm: 0 1px 2px rgba(46, 16, 101, 0.06);
+    --swh-shadow-md: 0 2px 8px rgba(46, 16, 101, 0.08), 0 1px 2px rgba(46, 16, 101, 0.04);
+    --swh-shadow-lg: 0 8px 24px rgba(46, 16, 101, 0.12), 0 2px 6px rgba(46, 16, 101, 0.06);
+
+    /* Typography */
+    font-family: 'Geist Sans', 'Geist', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)